x/projects/keeper/msg_server_add_project_keys.go

```diff
@@ -18,6 +18,13 @@ func (k msgServer) AddKeys(goCtx context.Context, msg *types.MsgAddKeys) (*types
 		)
 	}
 
+	if len(msg.ProjectKeys) > types.MAX_KEYS_AMOUNT {
```

Comment:
isnt this a double check since we already did it in k.AddKeysToProject?

Reply:
removed. see https://github.com/lavanet/lava/commit/df5a276856b7e6bdf5405bc56de276d195ddde62